chart/templates/mayastor/obs/stats-service.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.eventing.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-stats-aggregator
```

Comment:
Can we use just stats instead of stats aggregator?

Reply:
done